src/main/resources/templates/c_plus_plus/test/tests/TestCompile.py

```
@@ -0,0 +1,40 @@
+
+    target: str
+
+    def __init__(
+        self,
+        buildDir: str,
+        target: str = "all",
+        requirements: List[str] | None = None,
+        name: str = "TestCompile",
+    ) -> None:
+        super().__init__(
+            name, buildDir, "cmake", requirements, timeoutSec=10
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)